frontend/src/components/ConfigBar/components/PresetButtons.vue

```diff
@@ -115,14 +114,17 @@ export default {
     activeUser() {
       return this.$store.state.activeUser;
     },
+    simRunning() {
+      return this.$store.state.simRunning
```

Comment:
Some lines you have `;` some you don't, make sure you are consistent - seems you aren't running code formatter

Reply:
I'll do this once merged to main, to avoid conflicts